src/main/java/net/dungdealers/diapersplus/DiaperItem/DiaperItem.java

```diff
@@ -0,0 +1,28 @@
+package net.dungdealers.diapersplus.DiaperItem;
+
+import net.fabricmc.fabric.api.item.v1.FabricItemSettings;
+import net.fabricmc.fabric.api.itemgroup.v1.FabricItemGroupEntries;
+import net.fabricmc.fabric.api.itemgroup.v1.ItemGroupEvents;
+import net.minecraft.item.Item;
+import net.dungdealers.diapersplus.Diapers;
+import net.minecraft.item.ItemGroups;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.util.Identifier;
+
+public class DiaperItem {
```

Comment:
If this is going to be our main item class for the mod, we should give it a more general name like DPItems. A more specific class for the functionality of the diapers (I fucking typed this shit unironically) will be made later (watch episode 5 of the tutorial series for 'advanced items')